broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -1,25 +1,14 @@
 const { PublicError } = require('grpc-methods')
+
 const { convertBalance, Big } = require('../../utils')
-/**
- * @constant
- * @type {Long}
- * @default
- */
-const MINIMUM_FUNDING_AMOUNT = 400000
+const { currencies: currencyConfig } = require('../../config')
 
 /**
- * This is the max allowed balance for a channel for LND while software is currently
- * in beta
- *
- * Maximum channel balance (no inclusive) is 2^32 or 16777216
- * More info: https://github.com/lightningnetwork/lnd/releases/tag/v0.3-alpha
- *
- * @todo make this engine agnostic (non-LND)
  * @constant
  * @type {Long}
  * @default
  */
-const MAX_CHANNEL_BALANCE = 16777215
+const MINIMUM_FUNDING_AMOUNT = 400000
```

Comment:
What determines the minimum funding amount? Is it the same source as the max channel balance? If so why isn't this value also added to the configuration?

Reply:
@martineehrlich minimum funding amount is a value that we set for channels. This change was not made in the PR (it shows up as green because I removed `MAX_CHANNEL_BALANCE` constant